MADRaS/traffic/const_vel.py

```diff
@@ -2,17 +2,17 @@
 import sys
 import yaml
 import numpy as np
-from controllers.pid import PID
-from utils.gym_torcs import TorcsEnv
-import utils.snakeoil3_gym as snakeoil3
-from utils.madras_datatypes import Madras
+from MADRaS.controllers.pid import PID
+from MADRaS.utils.gym_torcs import TorcsEnv
+import MADRaS.utils.snakeoil3_gym as snakeoil3
+from MADRaS.utils.madras_datatypes import Madras
 
 madras = Madras()
 with open("./traffic/configurations.yml", "r") as ymlfile:
```

Comment:
There is a small issue is with this open statement. I needed to specify my complete path from home for each of these for the file to be found.

Reply:
yeah I kept testing from the MADRaS folder.